LayoutTests/imported/w3c/web-platform-tests/css/css-backgrounds/background-clip/clip-text-text-decorations.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>background-clip: text shows text decorations</title>
+<link rel="match" href="clip-rounded-corner-ref.html">
+<link rel="help" href="https://drafts.csswg.org/css-backgrounds-3/#corner-clipping">
+<style>
+    .test {
+        line-height: 2em;
+        font-size: 40px;
+        font-family: Ahem;
+        color: transparent;
+        background-color: green;
+        background-clip: text;
+        text-decoration-thickness: 20px;
+    }
+</style>
+<body>
+    <div class="test" style="text-decoration: underline">AAAA</div>
+    <div class="test" style="text-decoration: line-through">AAAA</div>
+    <div class="test" style="text-decoration: overline">AAAA</div>
```

Comment:
The `text-decoration-line` should be used here, otherwise, `text-decoration` will override the `text-decoration-thickness` value.

It works in Safari because `text-decoration-thickness` has not been added to the text-decoration shorthand.